### ISSUE TITLE
Crash on start

### ISSUE THREAD

**Issue by andresako** [bbl-team/Modpack-Support#6]
### Modpack

Stoneopolis

### Version

B4.1

### Describe the bug

Fresh install, New world created, when click on it to start, the game crash during loading screen. 
Tried to delete a re-create the new world. Tried to reinstall the modpack. Same output every time.

https://pastebin.com/sQinC7Ld

### Steps to reproduce

Trying to load the new world. 

Install modpack.
Start modpack.
Create new world.
Start the new world.
- Crash during loading -

### Expected behaviour

The world to load.

### Additional information

I played other BBL modpacks in the past with no problem

**Comment by b0bst3r** [comment]
Your pastebin link is blank can you upload again?

**Comment by andresako** [comment]
Try this one  https://pastebin.com/CfqTS0L7

**Comment by andresako** [comment]
https://pastebin.com/QVxbcGhd
Previous one was created from a guest account... not sure what happened. 
This one is from a registered account, it's on "pending moderation" 
I hope it get's approved 

sneak peak:

A fatal error has been detected by the Java Runtime Environment:

  SIGBUS (0xa) at pc=0x00000001154f14e4, pid=1356, tid=148199

 JRE version: OpenJDK Runtime Environment Microsoft-8035246 (17.0.8+7) (build 17.0.8+7-LTS)
Java VM: OpenJDK 64-Bit Server VM Microsoft-8035246 (17.0.8+7-LTS, mixed mode, tiered, compressed oops, compressed class ptrs, g1 gc, bsd-aarch64)
 Problematic frame:
 v  ~StubRoutines::SafeFetch32

 No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again

**Comment by b0bst3r** [comment]
No that one is also blocked in future use https://gist.github.com/ for pastes.

I've seen that error before are you by chance using MacOS?

**Comment by andresako** [comment]
https://gist.github.com/andresako/77a1d2a79eb3e1f59ffba2ae12f42bf6

Yep, I'm using MacOS

**Comment by Bebobobo** [comment]
you need to downgrade your java or switch distro. https://discord.com/channels/716249661798612992/1051803244554371183/1224168243984334970

**Comment by andresako** [comment]
woah... 
I'm an Android dev, I do need the latest for my work. 
How it's possible that I'm able to use for other modpacks and not this one?
ps, that link is not able to load, I can see the channel but not the comment.

**Comment by Bebobobo** [comment]
i dont know why it works for some and not others. another option is to change java distro

**Comment by andresako** [comment]
what java version should I use for this?

**Comment by Bebobobo** [comment]
this is what was recommended by b0bst3r. https://www.azul.com/

**Comment by Azulear** [comment]
Or downgrade to Oracle Java 17.0.6.
That should be the latest release without this bug on Mac.

**Comment by Bebobobo** [comment]
@Azulear read the comments

**Comment by andresako** [comment]
Awesome, it's working now!

Basically, installed the dmg.
Changed the java version to this one under /Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home/bin

And now everything works.
Thank you for your help @Bebobobo

**Comment by Bebobobo** [comment]
excellent. glad to help

**Comment by Azulear** [comment]
> @Azulear read the comments

Sry and what's the problem?
If you refer to the link you posted, then it is inconvenient to link to something that is not directly accessible.

**Comment by b0bst3r** [comment]
You may be interested @andresako the bug is related to this https://bugs.openjdk.org/browse/JDK-8307549 it was supposedly fixed in 17.0.9 for the initial problem.  As to why it works with some and not all modpacks is extremely mod specific, the initial problem was caused by mod Spark which is in Stoneopolis and why 17.0.7/8 doesn't work but 17.0.9 should.  I think as mods develop some other mods maybe causing the issue.  The most common fix I found that most MacOS users were doing was to switch to the Azul distro which doesn't have the issue at all.  We have had people on 17.0.10 with this issue so I think other mods are/maybe causing that.

**Comment by Bebobobo** [comment]
> > @Azulear read the comments
> 
> Sry and what's the problem? If you refer to the link you posted, then it is inconvenient to link to something that is not directly accessible.

i was linking the direct response within BBL chat for this error. You suggested to downgrade oracle when the op stated he could not due to his work environment. Thats why i said read the comments.

**Comment by moody69420** [comment]
U can have multiple javas installed and use different java per pack so no real issue there no need to uninstall at all.

**Comment by chr1sbates** [comment]
Just to confirm this is reportedly due to the Spark mod. Another work-around would have been removing the Spark mod. 

```
Java versions 17.0.7 and above have a [bug](https://bugs.openjdk.org/browse/JDK-8307549) on Apple Silicon/arm64/aarch64 Macs. The fix is to manually download an older version like Java 17.0.6 and use that.

I know the [Spark](https://github.com/lucko/spark/issues/339) mod uses code that triggers that bug.

Mojang recently updated Java in the Minecraft Launcher to use Java 17.0.8.

Edit: This has been fixed as of OpenJDK 17.0.9!
```

**Comment by CharZinta** [comment]
Thank you to all who have provided support those who needed support getting this working.
